src/includes/platforms/configuration/integrations/plugin.mdx

```diff
@@ -107,3 +107,21 @@ Available options:
 <PlatformContent includePath="configuration/reporting-observer" />
 
 </PlatformSection>
+
+<PlatformSection supported={["javascript"]}>
+
+### ContextLines
+
+_Import name: `Sentry.Integrations.ContextLines`_
+
+This integration attaches source code of inline JavaScript inside the page's HTML (e.g. JS in `<script>` tags) to errors captured by the SDK.
+Note that this integration _cannot_ collect source code from assets referenced by your HTML (e.g. `<script src="..." />`).
+
+Generally, the Sentry backend tries to attach source code to these errors by fetching the page and extracting the respective lines.
```

Comment:
```suggestion
This integration is useful when you have inline JS code in HTML pages that can't be accessed by Sentry's backend, for example, due to a login-protected page.

If Sentry can fetch the relevant JavaScript file this integration isn't needed. 
```

Reply:
I tried to combine the sentences here to be more concise and user-focused. Feel free to tweak if I have any details wrong.